src/error_handler.sh

```
@@ -0,0 +1,405 @@
+    ["CONFIG_NOT_FOUND"]=31
+    ["NETWORK_TIMEOUT"]=40
+    ["NETWORK_CONNECTION"]=41
+)
+
+# エラーメッセージ生成
+generate_error_message() {
+    local error_code="$1"
+    local context="${2:-}"
+    
+    case "$error_code" in
+        ${ERROR_CODES["GEMINI_CLI_NOT_FOUND"]})
```

Shellcheck (0.10.0)

[warning] Quote expansions in case patterns to match literally rather than as a glob.

(SC2254)